### PR TITLE
CDN tests can run on canary

### PR DESCRIPTION
**Pull request by mcardosos**
Thanks for your contribution! Please let us know what problems this PR addresses or what functionality it adds. Please link to other related issues and PRs to speed up review. Thank you!

### CODE REVIEW

File: cdn/cdn_test.go [hunk 1]
```diff
@@ -4,10 +4,20 @@ import (
...
 	"fmt"
 	"log"
+	"os"
+	"testing"
 
 	"github.com/Azure-Samples/azure-sdk-for-go-samples/helpers"
 )
 
+func TestMain(m *testing.M) {
+	err := helpers.ParseArgs()
```
Comment (marstr) [inline cdn/cdn_test.go]: Given that this ends up looking so much like the default implementation of TestMain, I'd add a comment here mentioning that ParseArgs allows us to target the particular region that we need.

Comment (marstr) [reply cdn/cdn_test.go]: Either that or we move away from command line args for this and move to environment variables for everything.

Comment (mcardosos) [reply cdn/cdn_test.go]: CLI is beautiful.

Comment (mcardosos) [reply cdn/cdn_test.go]: Fixed

**Pull request merged**